test/mitmproxy/data/har_files/charles.json

```diff
@@ -81,7 +81,7 @@
             "headers": [
                 [
                     "Content-Type",
-                    "text/html; charset=utf-8"
+                    "text/html"
```

Comment:
We no longer update the `Content-Type`, instead the value from the HAR file is used.